packages/core/src/spec/lib/TwaManifestSpec.ts

```diff
@@ -87,6 +88,7 @@ describe('TwaManifest', () => {
       expect(twaManifest.startUrl).toBe('/');
       expect(twaManifest.iconUrl).toBeUndefined();
       expect(twaManifest.maskableIconUrl).toBeUndefined();
+      expect(twaManifest.monochromeIconUrl).toBeUndefined();
```

Comment:
Could you add a testcase (or change an existing one) where this is not undefined?